cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Connectors.java

```diff
@@ -326,4 +326,14 @@ private static String addTLSConfigToMirrorMaker2ConnectorConfig(Map<String, Obje
             return "PLAINTEXT";
         }
     }
+
+    private static boolean hasMatchingBootstrapServers(List<KafkaMirrorMaker2ClusterSpec> clusterList, String connectClusterAlias, String targetClusterAlias) {
+        List<String> configs = clusterList
+                .stream()
+                .filter(cluster -> connectClusterAlias.equals(cluster.getAlias()) || targetClusterAlias.equals(cluster.getAlias()))
+                .map(KafkaMirrorMaker2ClusterSpec::getBootstrapServers)
+                .toList();
+
+        return configs.size() == 2 && configs.get(0).equals(configs.get(1));
```

Comment:
If you insist on doing it this comúplicated way despite my comments, maybe you can add at least some comment/JavaDoc to what it does and how.

Reply:
Apologies @scholzj, I missed that earlier comment. Did you mean something like this?

```
private static boolean hasMatchingBootstrapServers(List<KafkaMirrorMaker2ClusterSpec> clusterList, String connectClusterAlias, String targetClusterAlias) {
    // Find the cluster for the connectClusterAlias
    String connectClusterBootstrap = clusterList.stream()
            .filter(cluster -> connectClusterAlias.equals(cluster.getAlias()))
            .map(KafkaMirrorMaker2ClusterSpec::getBootstrapServers)
            .findFirst()
            .orElse(null);

    // Find the cluster for the targetClusterAlias
    String targetClusterBootstrap = clusterList.stream()
            .filter(cluster -> targetClusterAlias.equals(cluster.getAlias()))
            .map(KafkaMirrorMaker2ClusterSpec::getBootstrapServers)
            .findFirst()
            .orElse(null);

    // Return true if both are found and have matching bootstrap servers
    return connectClusterBootstrap != null && connectClusterBootstrap.equals(targetClusterBootstrap);
}

```

Let me know if this is what you were suggesting.

Reply:
Yes, this is what I would have used.

Reply:
Sure. I have pushed in the latest changes. Please can you re-review.